scripts/management/commands/secret_santa.py

```diff
@@ -24,31 +24,41 @@ def handle(self, *args, **options):
         for count, user in enumerate(users):
             to_from_dict['Par %s' % (count+1)] = {'To': user.user.get_full_name(),
                                                   'From': users[random_users[count]].user.get_full_name()}
-
             self.send_mail(user, users[random_users[count]])
 
         self.write_to_file(to_from_dict)
-        self.mail_to_committee(to_from_dict)
+        self.mail_to_committee()
 
     # Function for generating a list with random numbers corresponding to
     # the indexes of the list of users attending the event.
     @staticmethod
     def generate_random_list(users):
         to_from = random.sample(range(len(users)), len(users))
-        while(not Command.check_random_list(to_from)):
-            to_from = random.sample(range(len(users)), len(users))
+        Command.check_random_list(to_from)
```

Comment:
The `check_random_list` function mutates the to_from list and returns the updates list. While the current implementation _works_, it's better practice to assign the returned value from the function to a variable. This makes the code less magic. :-)

You could also copy (`to_from[:]`) the list to make sure you're not mutating the list, but rather working on a copy of it.

If you did that here, you would get different results, since you don't store the results of the function but rather expect it to mutate the referenced list.